templates/mount.erb

```diff
@@ -33,7 +33,7 @@ Conflicts=<%= @mnt_conflicts.join(' ') %>
 Conflicts=<%= @mnt_conflicts %>
 <% end -%>
 <% end -%>
-<% if @mnt_default_dependencies -%>
+<% if @mnt_default_dependencies == false -%>
```

Comment:
This is dependent on the upstream default.  I'd prefer to see this as a "if not null" so the line is included whether true or false.